### PR TITLE
en,zh: add v1.5.2 release notes

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.5.2.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: TiDB Operator 1.5.2 Release Notes
+---
+
+# TiDB Operator 1.5.2 Release Notes
+
+Release date: January 19, 2023
+
+TiDB Operator version: 1.5.2
+
+## New features
```
Comment (BornChanger) [inline en/releases/release-1.5.2.md:11]: From v1.5.2, TiDB operator supports back up and restore the data of a TiDB cluster deployed across multiple AWS Kubernetes clusters to AWS storage using EBS volume snapshots ([#5003](https://github.com/pingcap/tidb-operator/pull/5003), [@BornChanger](https://github.com/BornChanger), [@WangLe1321](https://github.com/WangLe1321), [@YuJuncen](https://github.com/YuJuncen), [@csuzhangxc](https://github.com/csuzhangxc)). For detailed information, visit [Back up Data Using EBS Snapshots across Multiple Kubernetes](backup-by-ebs-snapshot-across-multiple-kubernetes.md) and [Restore Data Using EBS Snapshots across Multiple Kubernetes](restore-from-ebs-snapshot-across-multiple-kubernetes.md)

File: zh/releases/release-1.5.2.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: TiDB Operator 1.5.2 Release Notes
+---
+
+# TiDB Operator 1.5.2 Release Notes
+
+发布日期: 2024 年 1 月 19 日
+
+TiDB Operator 版本：1.5.2
+
+## 新功能
```
Comment (BornChanger) [inline zh/releases/release-1.5.2.md:11]: 从 1.5.2 版本开始, TiDB operator 支持基于 AWS EBS 快照的备份能力的跨多个 K8S 集群的备份恢复.([#5003](https://github.com/pingcap/tidb-operator/pull/5003), [@BornChanger](https://github.com/BornChanger), [@WangLe1321](https://github.com/WangLe1321), [@YuJuncen](https://github.com/YuJuncen), [@csuzhangxc](https://github.com/csuzhangxc)). 访问文档[Back up Data Using EBS Snapshots across Multiple Kubernetes](https://github.com/pingcap/docs-tidb-operator/pull/2485/backup-by-ebs-snapshot-across-multiple-kubernetes.md) 和 [Restore Data Using EBS Snapshots across Multiple Kubernetes](https://github.com/pingcap/docs-tidb-operator/pull/2485/restore-from-ebs-snapshot-across-multiple-kubernetes.md)可以了解详情.

File: en/releases/release-1.5.2.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Improvements
+
+- `startScriptVersion: v2` supports waiting for Pod IP to match the one published to external DNS before starting a PD or TiKV to better support scenarios such as Stale Read ([#5381](https://github.com/pingcap/tidb-operator/pull/5381), [@smineyev81](https://github.com/smineyev81))
+- `startScriptVersion: v2` supports explicitly specifying PD addresses to better support scenarios where a TiDB cluster is deployed across Kubernetes ([#5400](https://github.com/pingcap/tidb-operator/pull/5400), [@smineyev81](https://github.com/smineyev81))
+- tidb-operator Helm Chart supports specify resource lock for leader election when deploying Advanced-StatefulSet ([#5448](https://github.com/pingcap/tidb-operator/pull/5448), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (ran-huang) [inline en/releases/release-1.5.2.md:17]:
```suggestion
- tidb-operator Helm Chart supports specifying resource lock for leader election when deploying Advanced StatefulSet ([#5448](https://github.com/pingcap/tidb-operator/pull/5448), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.5.2.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Bug fixes
+
+- Fix the issue that changing meta information such as annotations and replacing volumes at the same time may cause a deadlock for TiDB Operator reconcile ([#5382](https://github.com/pingcap/tidb-operator/pull/5382), [@anish-db](https://github.com/anish-db))
```
Comment (ran-huang) [inline en/releases/release-1.5.2.md:21]:
```suggestion
- Fix the issue that changing meta information such as annotations and replacing volumes at the same time might cause a deadlock for TiDB Operator reconcile ([#5382](https://github.com/pingcap/tidb-operator/pull/5382), [@anish-db](https://github.com/anish-db))
```

File: en/releases/release-1.5.2.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- Fix the issue that the PD member might be set the wrong label when replacing volumes ([#5393](https://github.com/pingcap/tidb-operator/pull/5393), [@anish-db](https://github.com/anish-db))
```
Comment (ran-huang) [inline en/releases/release-1.5.2.md:22]:
```suggestion
- Fix the issue that the PD member might be set to the wrong label when replacing volumes ([#5393](https://github.com/pingcap/tidb-operator/pull/5393), [@anish-db](https://github.com/anish-db))
```

File: zh/releases/release-1.5.2.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+title: TiDB Operator 1.5.2 Release Notes
+---
+
+# TiDB Operator 1.5.2 Release Notes
+
+发布日期: 2024 年 1 月 19 日
```
Comment (ran-huang) [inline zh/releases/release-1.5.2.md:7]:
```suggestion
发布日期：2024 年 1 月 19 日
```

File: zh/releases/release-1.5.2.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+# TiDB Operator 1.5.2 Release Notes
+
+发布日期：2024 年 1 月 19 日
+
+TiDB Operator 版本：1.5.2
+
+## 新功能
+
+从 v1.5.2 起, TiDB operator 支持基于 AWS EBS 快照的备份能力的跨多个 K8S 集群的备份恢复。更多详情，请查看文档 [Back up Data Using EBS Snapshots across Multiple Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/backup-by-ebs-snapshot-across-multiple-kubernetes) 和 [Restore Data Using EBS Snapshots across Multiple Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/restore-from-ebs-snapshot-across-multiple-kubernetes)。([#5003](https://github.com/pingcap/tidb-operator/pull/5003), [@BornChanger](https://github.com/BornChanger), [@WangLe1321](https://github.com/WangLe1321), [@YuJuncen](https://github.com/YuJuncen), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (ran-huang) [inline zh/releases/release-1.5.2.md:13]:
```suggestion
从 v1.5.2 起, TiDB Operator 支持基于 AWS EBS 快照的备份能力的跨多个 K8S 集群的备份恢复。更多详情，请查看文档 [Back up Data Using EBS Snapshots across Multiple Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/backup-by-ebs-snapshot-across-multiple-kubernetes) 和 [Restore Data Using EBS Snapshots across Multiple Kubernetes](https://docs.pingcap.com/tidb-in-kubernetes/stable/restore-from-ebs-snapshot-across-multiple-kubernetes)。([#5003](https://github.com/pingcap/tidb-operator/pull/5003), [@BornChanger](https://github.com/BornChanger), [@WangLe1321](https://github.com/WangLe1321), [@YuJuncen](https://github.com/YuJuncen), [@csuzhangxc](https://github.com/csuzhangxc))
```

File: en/releases/release-1.5.2.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+# TiDB Operator 1.5.2 Release Notes
+
+Release date: January 19, 2023
+
+TiDB Operator version: 1.5.2
+
+## New features
+
+Starting from v1.5.2, TiDB Operator supports backing up and restoring the data of a TiDB cluster deployed across multiple AWS Kubernetes clusters to AWS storage using EBS volume snapshots. For more information, refer to [Back up Data Using EBS Snapshots across Multiple Kubernetes](backup-by-ebs-snapshot-across-multiple-kubernetes.md) and [Restore Data Using EBS Snapshots across Multiple Kubernetes](restore-from-ebs-snapshot-across-multiple-kubernetes.md). ([#5003](https://github.com/pingcap/tidb-operator/pull/5003), [@BornChanger](https://github.com/BornChanger), [@WangLe1321](https://github.com/WangLe1321), [@YuJuncen](https://github.com/YuJuncen), [@csuzhangxc](https://github.com/csuzhangxc))
```
Comment (ran-huang) [inline en/releases/release-1.5.2.md:13]:
```suggestion
Starting from v1.5.2, TiDB Operator supports backing up and restoring the data of a TiDB cluster deployed across multiple AWS Kubernetes clusters to AWS storage using EBS volume snapshots. For more information, refer to [Back up Data Using EBS Snapshots across Multiple Kubernetes](../backup-by-ebs-snapshot-across-multiple-kubernetes.md) and [Restore Data Using EBS Snapshots across Multiple Kubernetes](../restore-from-ebs-snapshot-across-multiple-kubernetes.md). ([#5003](https://github.com/pingcap/tidb-operator/pull/5003), [@BornChanger](https://github.com/BornChanger), [@WangLe1321](https://github.com/WangLe1321), [@YuJuncen](https://github.com/YuJuncen), [@csuzhangxc](https://github.com/csuzhangxc))
```

**Pull request merged**